pkg/lint/support/message.go

```diff
@@ -63,6 +61,11 @@ func (m Message) String() string {
 
 // Returns true if the validation passed
 func (l *Linter) RunLinterRule(severity Severity, lintError LintError) bool {
+	// severity is out of bound
+	if severity < 0 || int(severity) >= len(sev) {
```

Comment:
I can't `invalid operation: severity >= len(sev) (mismatched types Severity and int) ` maybe I should remove the severity type